wedlocks/src/__tests__/handler.test.js

```diff
@@ -16,12 +18,14 @@ describe('handler', () => {
           throw error
         }
         expect(response.statusCode).toBe(204)
-        done()
+        new Promise((done) => {
+          done()
+        })
```

Comment:
Instead of this. Use promise resolve to resolve the response value and capture it using variable.

This way, you can remove the expect and assertions outside this callback.